modal/functions.py

```diff
@@ -203,7 +207,7 @@ async def create(function_id: str, args, kwargs, client: _Client):
 
         function_call_id = response.function_call_id
 
-        item = await _create_input(args, kwargs, client)
+        item = await _create_input(args, kwargs, client, final_input=True)
```

Comment:
Note: I'm not sure if this change is correct. See Elias's comment on Slack: https://modal-com.slack.com/archives/C056CGAANRM/p1707123558333609?thread_ts=1706890751.262849&cid=C056CGAANRM

The reason I need this is I need the entire `_container_entrypoint.py` command to exit in order to signal the client that the PTY shell has closed.

If we do not want to rely on `final_input=True`, we will have to find another way to detect when the PTY shell has closed, perhaps by detecting when one input is finished.

I think it would be even better to detect when `breakpoint()` finishes rather than when the PTY shell has closed, but I don't know how to do this.

Reply:
We should not set `final_input=True` for all invocations. This would mean that 
calls like this would not reuse a container when they should:

```python
model = MyModel()
model.inference.remote(...)
model.inference.remote(...)
```